scheduler/src/cook/mesos/sandbox.clj

```diff
@@ -150,19 +136,38 @@
           (contains? #{:success :threshold} reason)
           (update :host->consecutive-failures dissoc hostname)))
 
+(defn- filter-task-ids-with-sandbox
```

Comment:
Nitpick: Consider renaming to `remove-task-ids-with-sandbox`

Reply:
Done.